### DISCUSSION TITLE
Conflicting "Camera specific" instructions for reolink vs. "preset" instructions for reolink

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#10573] [General]
In the [camera specific](https://docs.frigate.video/configuration/camera_specific/) docs for reolink it says to use the http/flv stream and to set input_args as `preset-rtsp-restream`. However in the [presets documentation](https://docs.frigate.video/configuration/ffmpeg_presets/) there is a `preset-http-reolink` described as "Only for reolink http, not when restreaming as rtsp". Which is best to use for reolink http/flv?

**Comment by NickM-27** [comment]
The recommended reolink config is what is recommended. These are not conflicting, the presets simply provides an option for connecting directly to a reolink camera if desired
